pavelib/assets.py

```diff
@@ -191,31 +221,59 @@ def compile_sass(options):
     """
     Compile Sass to CSS.
     """
+
+    # Note: import sass only when it is needed and not at the top of the file.
```

Comment:
interesting workaround. i'm not sure i like it but can live with it.... would this be moved up sometime later?